packages/devtools_app/test/flutter/debugger_screen_test.dart

```diff
@@ -114,15 +114,20 @@ void main() {
     });
 
     group('ConsoleControls', () {
-      testWidgets('Console Controls are not rendered when stdio is empty',
+      testWidgets('Console Controls are rendered disables when stdio is empty',
```

Comment:
Typo in "disables" should be: "disabled"

Reply:
will fix